src/yz_events.erl

```diff
@@ -106,6 +133,17 @@ terminate(_Reason, _S) ->
 %%% Private
 %%%===================================================================
 
+%% @doc Called by {@link yz_general_sup} to create the p ETS table used to
+%% track registered events. Created when we add the handler after the supervisor
+%% is already up for yz_events.
+-spec create_table() -> ok.
+create_table() ->
+    _ = ets:new(?MODULE, [named_table, public, set,
```

Comment:
It's been a few weeks since I wrote this, but I remember `protected` not working here. When you review @jvoegele / @fadushin, you can try and see if it works... update the code, yay :dancer: